tests/std/tests/P1132R7_out_ptr/test.cpp

```diff
@@ -75,9 +75,22 @@ void test_shared_ptr() {
         assert(count == 1);
         assert(*int_ptr == 32);
     }
+    // LWG-3734 Inconsistency in inout_ptr and out_ptr for empty case
+    {

```

Comment:
Nitpick: It would be nice to have a newline before this comment, to make it a bit clearer which block it applies to, and for consistency with right line 147. Occurs below on right line 134.